### ISSUE TITLE
[Question] How to build wiilove?

### ISSUE THREAD

**Issue by shazz** [HTV04/wiilove#5]
Hi,

I tried to recompile wiilove for the gamecube but there is a dependency on `luajit`, where can I find this library?

Thanks!

**Comment by shazz** [comment]
Ah I guess this is this one https://github.com/HTV04/LuaJIT :)

**Comment by shazz** [comment]
Yeah I did recompile for the GC LuaJIT, grrlib-mod, png-mod and finally Wiilove... sorry Cubelove :) but got a black screen when running the wiilove.dol (but I did not know what to expect)

I'll go back and check first the deps I recompiled still work :)

Happy to help !

**Comment by HTV04** [comment]
Just wondering, are you testing this on real hardware, and if so, do you have an SD Gecko? Unfortunately Dolphin can't emulate those at the moment, but if you have one, in theory you should be able to store and load WiiLÖVE games from it.

**Comment by shazz** [comment]
Yeah I'm only testing on a real GameCube, I found Dolphin pretty...inaccurate, most of my stuff just doesn't work in Dolphin.

To debug I have a USB Gecko but I prefer to use the BBA, faster and more relaxe, I wrote a simple network launcher / debugger (similar to PS2Link).

So I tested luaJIT with a simple API example, it works on my GC, so I guess I broke something on Wiilove.
